docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,167 @@
+
+* You control the cadence of change for installed workload versions. The alternative mode of operation without using workload sets is called *loose manifests update mode*. In this mode, workloads are updated automatically as new versions of individual workloads are released onto any configured NuGet feeds. In workload set update mode, workloads stay at a specific workload set version until you explicitly change that version.
+* You can install and update a combination of workload versions that ship at the same time and are known to work together.
+* You can ensure that everyone on your team is always working on the same workload versions.
+* You don't have to use a rollback file to specify what workload version you want to be on.
+
+Here are some ways you can use workload sets:
+
+* "Pin" the `install` command to a specific workload-set version.
+* Update installed workloads to the latest available workload-set version.
+* Update to a specified workload-set version.
+* Specify the workload set version in global.json.
+* Check your current update mode and workload-set version.
+
+And you can still choose to install, update, or restore to the latest version of each individual workload, ignoring workload sets.
+
+## Prerequisites
+
+* [.NET 8.0.400 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later.
+
+  In 8.0.400 SDK, the `dotnet workload` commands are in workload-set update mode only when it's explicitly selected. Starting in [.NET 9 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/9.0), `dotnet workload` commands are in workload-set update mode by default.
+
+## "Pin" the install command
+
+A `dotnet workload install` command with the `--version` option "pins" the `install` command in workload set update mode with the specified workload set version.
+<!--todo: install without --version in workload set update mode pins to latest workload set?-->
+The command no longer automatically installs the newest workload based on loose manifests. For example:
+
+1. ```dotnetcli
+   dotnet workload install aspire --version 9.0.100-preview.7.24414.1
+   ```
+
+   When this command runs:
+
+   * It selects workload set update mode if that isn't already selected.
+   * It gets the workload set that has the specified version.
+   * From the workload set, it gets the manifest version of the specified workload.
+   * It installs the manifest version of the workload.
+   * It stays in workload-set update mode when it's done.
+
+1. ```dotnetcli
+   dotnet workload install maui-ios
+   ```
+
+   This command installs the workload set version `9.0.100-preview.7.24414.1`, since the preceding `install` command example pinned that workload set.
```

Comment:
```suggestion
   This command installs the the `maui-ios` workload using the workload version from the workload set version `9.0.100-preview.7.24414.1`, since the preceding `install` command example pinned that workload set.
```

I think it's worth clarifying where the version of the workload being installed are coming from a bit like you did in the `aspire` example above.